src/frontend/src/hooks/use-telescope-contributor.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+
+const useTelescopeContributor = () => {
+  const [contributor, setContributor] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    const fetchContributors = async () => {
+      setLoading(true);
+      const response = await fetch(
```

Comment:
You *must* wrap all `await`'ed code in a try/catch, since network requests like this can fail for many reasons.

However, I'm going to ask you to go another route and use the [`useSWR`](https://swr.vercel.app/) hook to do this.  It will handle errors and caching for us.

Reply:
I'm having a hard time getting SWR to work. I have tried many different approaches but nothing seems to work. I saw that someone else used it in Posts, and I tried doing exactly what they did but with the GitHub url and that still didn't work.